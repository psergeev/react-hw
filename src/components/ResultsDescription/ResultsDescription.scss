.results-description {
    background-color: #F5F5F5;
    height: 40px;
    line-height: 40px;
    padding: 0 50px;
    color: #666;
}
