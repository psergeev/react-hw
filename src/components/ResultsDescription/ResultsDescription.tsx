import * as React from 'react';
import './ResultsDescription.scss';

interface Props {
    genre: string;
}

export default React.memo((props: Props) => (
    <div className="results-description">{`Films by ${props.genre} genre`}</div>
));
