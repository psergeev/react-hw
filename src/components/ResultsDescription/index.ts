import ResultsDescription from './ResultsDescription';

export default ResultsDescription;
