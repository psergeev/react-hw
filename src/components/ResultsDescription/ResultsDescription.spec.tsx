import * as React from 'react';
import * as renderer from 'react-test-renderer';
import ResultsDescription from './ResultsDescription';

test('ResultsDescription component works as expected', () => {
    const component = renderer.create(
        <ResultsDescription
            genre="Fiction"
        />
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
