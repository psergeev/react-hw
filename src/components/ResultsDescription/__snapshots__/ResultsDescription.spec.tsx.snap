// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsDescription component works as expected 1`] = `
<div
  className="results-description"
>
  Films by Fiction genre
</div>
`;
